Wurstpack/wurstscript/lib/data structures/LinkedList.wurst

```diff
@@ -151,8 +151,8 @@ public class LLIterator<Q>
 		if current != dummy
 			current.next.prev = current.prev
 			current.prev.next = current.next
-			current = current.prev
 			destroy current
+			current = current.prev
```

Comment:
you should not use an object after it is destroyed. This might give compilation errors in future versions.